tf_helpers/lax.py

```diff
@@ -0,0 +1,465 @@
+# Copyright 2020 The Google/TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+import builtins
+from typing import NamedTuple, Sequence
+import string
+import numpy as onp
+from tensorflow.python.ops import numpy_ops as np
+import tensorflow as tf
+from tensorflow import nn
+import sys
+
+_max = builtins.max
+
+
+def conv_shape_tuple(lhs_shape, rhs_shape, strides, pads, batch_group_count=1):
```

Comment:
It appears that this and several other pure python functions are the same as in `lax` - should we prefer to just import and use them instead, especially if our goal is to mirror the lax API and stay up to date if it changes?